patches/aspnetcore/0008-Add-missing-reference.patch

```diff
@@ -0,0 +1,24 @@
+From 1e210f2449b0e9907d34c5d658405f4775a9b2c4 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Fri, 16 Aug 2019 15:55:31 -0500
+Subject: [PATCH 8/8] Add missing reference
+
+---
+ .../src/Microsoft.AspNetCore.Connections.Abstractions.csproj     | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj b/src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj
+index 52711dccf0..960e572814 100644
+--- a/src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj
++++ b/src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj
+@@ -14,6 +14,7 @@
+     <Reference Include="Microsoft.AspNetCore.Http.Features" />
+     <Reference Include="Microsoft.Extensions.ActivatorUtilities.Sources" />
+     <Reference Include="System.IO.Pipelines" />
++    <Reference Include="System.Threading.Tasks.Extensions" />
```

Comment:
Hmm This seems suspicious. Why is this needed?

Reply:
Yeah, I tried this because of this error:
` ConnectionContext.cs(18,47): error CS0012: The type 'ValueTask' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Threading.Tasks.Extensions, Version=4.2.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. [/home/chris/tarball/src/AspNetCore.4a42afc5aea63750638e118560d43db04bd9ccc2/src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.     csproj]`

but adding it actually caused the production build to fail.  Any idea what's going on here?

On source-build, we've committed to minimal patches, but as long as they're in-flight and will make it into the next release, we'll be including those as a separate "temporary patch" statistic, so no worries as long as we can get the ball rolling.

Reply:
Actually this is almost definitely caused by DanS's comment below.  Trying it out.

Reply:
That didn't seem to fix the issue.  @dseefeld, do you know if there's any issues with our reference version of System.Threading.Tasks.Extensions right now?  I can see it's being included as a reference, the versions match, and it does include `ValueTask`.

Reply:
@crummel  I don't know of any issues.  You could remove just that reference-package from the tarball reference-packages/packages directory to see if that narrows it down to a problem with that package.

Reply:
Our ref version of this package apparently has a public key problem, removing it as a workaround for now and logged https://github.com/dotnet/source-build/issues/1186.